ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-commands.sh

```diff
@@ -230,13 +269,22 @@ if [[ "${APPLY_ALL_SUBNETS}" == "no" ]]; then
         compute_subnets_len=$(cat "${vpc_info_file}" | jq -c -r --arg z "${zone}" '[.subnets | .[] | select(.zone.name==$z) | select(.name|test("compute"))] | length')
         compute_idx=$(( RANDOM % compute_subnets_len))
       ;;
+      "2Paging")
+        cp_idx="0"
+        compute_idx="0"
+      ;;
       *)
-        echo "unsupported value for PICKUP_SUBNETS_IN_ORDER"
+        echo "unsupported value for PICKUP_SUBNETS_ORDER"
         exit 2
       esac
+
       cat "${vpc_info_file}" | jq -c -r --arg z "${zone}" --argjson idx "$cp_idx" '[[.subnets | sort_by(.created_at) | .[] | select(.zone.name==$z) | select(.name|test("control-plane")) | .name][$idx]]' | yq-go r -P - >>${workdir}/controlPlaneSubnets.yaml
       cat "${vpc_info_file}" | jq -c -r --arg z "${zone}" --argjson idx "$compute_idx" '[[.subnets | sort_by(.created_at) | .[] | select(.zone.name==$z) | select(.name|test("compute")) | .name][$idx]]' | yq-go r -P - >>${workdir}/computerSubnets.yaml
     done
+
+    if [[ $PICKUP_SUBNETS_ORDER == "2Paging" ]]; then
+        ((checkUsedSubnets "${workdir}/controlPlaneSubnets.yaml" "${resource_group}") && (checkUsedSubnets "${workdir}/computerSubnets.yaml" "${resource_group}")) || exit 1
```

Comment:
The logic is wrong. Per your code, if `checkUsedSubnets "${workdir}/controlPlaneSubnets.yaml" "${resource_group}"` return non-zero, it will exit directly.

Reply:
if checkUsedSubnets fails, not cover 2Paging test precondition, so exit

Reply:
I did not get your point, per my reading of `checkUsedSubnets`, it is checking is the subnets in `used_subnets_file` on the 2nd page, if it is, then return pass, or else, return fail.

In your test case, once any subnet is used in OCP installation, then it covered your test case.

Reply:
> has cp subnet on the 2nd page to cover OCPBUGS-36236

I coped the description from the bug:
```
Use Bring Your Own Network (BYON) configuration (with Subnet names for CP and/or Compute) in install-config.yaml
```

So if only Compute subnets show up in the install-config.yaml, would meet its requirement.

Reply:
Per my understanding, for OCPBUGS-36236, cp subnet or compute subnet, at least one subnet can meet the requirement, cp subnet is not a MUST.
for OCPBUGS-36185, it is better to use compute subnet for testing from bug description, but per [discussion](https://redhat-internal.slack.com/archives/CF8SMALS1/p1731395051984239) with cloud team, actually cp nodes are also managed by MAPI. And I think OCPBUGS-36236 happened prior to  OCPBUGS-36185.

So if you prefer to check both cp and compute subnets to be more strict, I have no strong objection. You can keep the lines.